src/mds/MDBalancer.h

```diff
@@ -74,7 +74,7 @@ class MDBalancer {
 
   void handle_mds_failure(mds_rank_t who);
 
-  int dump_loads(Formatter *f) const;
+  int dump_loads(Formatter *f, const cmdmap_t &cmdmap) const;
```

Comment:
as suggested above, please pass in only an `int` value of the `depth` to this function and not the entire `cmdmap`